workflow-controller/rockcraft.yaml

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: workflow-controller
+summary: An image for Argo Workflows controller.
+description: |
+    This image is used as part of the Charmed Kubeflow product. An Argo workflow controller is a process that conforms to a specific interface that allows Argo to perform certain actions like monitoring pod logs, collecting artifacts, managing container lifecycles, etc.
+version: v3.3.8_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+build-base: ubuntu:22.04
+base: bare
+services:
+  argoexec:
```

Comment:
```suggestion
  workflow-controller:
```

Reply:
Ooopss.... there will be a lot of those with cut/pasting rock parts.

Reply:
haha yeah we'll all have at least a few :D